README.md

```diff
@@ -1,2 +1,82 @@
 # queries
 Hosts reusable log queries which are built into a single queries.json file.
+
+
+# Query database structure
+
+```yaml
+queries:
+  query1:  # that is query id (sorted by id)
```

Comment:
I'm not a big fan of it for the sake of keep it ordered, it should have another way, but I won't block it.